app/templates/_snippets.html

```diff
@@ -80,33 +80,114 @@
-              <td>{{ resource.format }}</td>
-              <td class="download truncate">
-                  <a href="{{ datapackageUrl | replace("/datapackage.json","")}}/{{ resource.path }}" class="explore">
-                      Download
-                  </a>
-              </td>
-            </tr>
-          {% endfor %}
-          </tbody>
-        </table>
+      <ul class="nav nav-pills">
+        <li class="active"><a data-toggle="pill" href="#download">Get data</a></li>
+        <li><a data-toggle="pill" href="#r">R</a></li>
+        <li><a data-toggle="pill" href="#pandas">Pandas</a></li>
+      </ul>
+
+      <div class="tab-content">
+        <div id="download" class="tab-pane fade in active">
+          <h2>DATA FILES</h2>
+          <div class="resource-listing part">
+            <table class="table table-bordered table-striped resource-listing">
+              <thead>
+                <th>Name</th>
+                <th>Format</th>
+                <th>Link</th>
+              </thead>
+              <tbody>
+              {% for resource in dataset.resources %}
+                <tr>
+                  <td><i class="fa fa-file-text-o"></i> <a href="#resource-{{resource.name}}" class="explore">{{resource.name}}</a></td>
+                  <td>{{ resource.format }}</td>
+                  <td class="download truncate">
+                      <a href="{{ datapackageUrl | replace("/datapackage.json","")}}/{{ resource.path }}" class="explore">
+                          Download
+                      </a>
+                  </td>
+                </tr>
+              {% endfor %}
+              </tbody>
+            </table>
+          </div>
+        </div>
+
+        <div id="r" class="tab-pane fade">
+          <h2>Using in R</h2>
+          <div class="part">
+            <p>In order to use Data Package in R follow instructions below:</p>
+            <pre>
+              <code>
+install.packages("devtools")
+
+library(devtools)
+install_github("hadley/readr")
+install_github("ropenscilabs/jsonvalidate")
+install_github("ropenscilabs/datapkg")
+
+#Load client
+library(datapkg)
+
+#Get Data Package
+datapackage &lt;- datapkg_read("https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest")
+
+#Package info
+print(datapackage)
+
+#Open actual data in RStudio Viewer
+{% for resource in dataset.resources %}
+View(datapackage$data$"{{ resource.name }}")
+{% endfor %}
+              </code>
+            </pre>
+          </div>
+        </div>
+
+        <div id="pandas" class="tab-pane fade">
+          <h2>Using in Pandas</h2>
+          <div class="part">
+            <p>In order to work with Data Packages in Pandas you need to install our packages:</p>
+            <pre>
+              <code>
+$ pip install datapackage
+$ pip install jsontableschema-pandas
+              </code>
```

Comment:
@zelima we're planning to remove flask-markdown sometimes in the future. We could do it somewhere in the logic using our markdown processor, but there is no need to do so as these instructions are fixed so we can have them in templates.